backend/app/dashboard/templates/dashboard/instructor-dashboard.html

```diff
@@ -0,0 +1,55 @@
+{% load static %}
+<!DOCTYPE html>
+<html>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.4/Chart.js"></script>
+    <script src="{% static '/prism.js' %}"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/wordcloud2.js/1.2.2/wordcloud2.min.js"></script>
+    <link href="{% static '/prism.css' %}" rel="stylesheet" />
+    <script>
+        function request_access(course_name, exercise_slug){
+            var id = "item-" + exercise_slug.replace(/\//g, '-');
+            if (document.getElementById(id)){
+                document.getElementById(id).remove();
+                return;
+            }
+            var item_div = document.createElement("div");
+
+           document.getElementById(exercise_slug).append(item_div);
+           item_div.id = id;
+            $.ajax({
```

Comment:
Se o jquery for só para o ajax, talvez seja melhor usar a função fetch para não precisar adicionar uma dependência externa.

Reply:
Isso aqui é mais porque eu não estava conseguindo adicionar o componente do exercicio de outra forma, vou tentar de novo.

Reply:
Mudei a lógica da request e removi as partes do ajax.